### PR TITLE
Add support for the events API.

### PR DESCRIPTION
**Pull request by amyboyd**
The HubSpot documentation for this is at: https://developers.hubspot.com/docs/methods/enterprise_events/http_api

### CODE REVIEW

**Comment by fungku** [comment]
I will look at this this week

File: src/Api/Events.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+     *
+     * @param  string $hubId Your HubSpot portal ID ("Hub ID"). You can find your Hub ID in the
+     *                       footer of the HubSpot UI, or in the URL. For example, in this URL:
+     *                       "https://app.hubspot.com/reports/56043/events/" your Hub ID is "56043".
+     * @param  string $eventId
+     * @param  string $contactEmail Optional.
+     * @param  float  $contactRevenue Optional - the monetary value this event means to you.
+     * @param  array  $contactProperties Optional - array of new contact properties.
+     */
+    public function addEvent($hubId, $eventId,
```
Comment (fungku) [inline src/Api/Events.php]: Can we rename `addEvent` to `trigger` so as not to be confused with creating a new event?

and matches hubspot's language

> If you're using HubSpot Events on the Enterprise version of the HubSpot software, you can use this call to **trigger** any custom events that you define in HubSpot (under "Reports -> Events"). You can then use events that you **trigger** to create smart lists, score leads and email contacts.

**Comment by lordthorzonus** [comment]
Hi whats the status of this being merged? Would love this functionality :+1: .

**Comment by amyboyd** [comment]
I'll try make the suggested changes tonight or tomorrow. Sorry about the
slow response, I've been busy :(

**Comment by amyboyd** [comment]
@RyanWinchester I've renamed `addEvent` to `trigger` , and rebased on master.

**Comment by RyanWinchester** [comment]
Thanks!

**Pull request merged**